cmd/admin-user-add.go

```diff
@@ -108,17 +128,27 @@ func mainAdminUserAdd(ctx *cli.Context) error {
 
 	console.SetColor("UserMessage", color.New(color.FgGreen))
 
+	stdin := ctx.Bool("stdin")
+
 	// Get the alias parameter from cli
 	args := ctx.Args()
 	aliasedURL := args.Get(0)
 
+	secretKey := ""
+	if stdin {
+		fmt.Scanln(&secretKey)
```

Comment:
We don't need to support stdin reading, it really doesn't make sense from a security point of view. We can simply have prompt and no prompt if secretKey was provided on the command line. 

We should avoid a breaking change here IMO @sinhaashish for example 

Old style is still honored
```
mc admin user add myminio foobar foo12345 writeonly
``` 

Let's say user didn't any inputs, then user should use interactive mode to start the prompt `mc admin user add -i` which starts the prompt
```
mc admin user add -i
Username: foobar
Policy: writeonly
Password: *****
```

This needs to be confirmed with @abperiasamy which I will let you know soon.

Reply:
:+1:

Reply:
Also, if we do that we should read a secret from the command line using [`ReadPassword`](https://godoc.org/golang.org/x/crypto/ssh/terminal#Terminal.ReadPassword).

Reply:
`cat ~/my_protected_password | mc admin user add --stdin` where ~/my_portected_password is readable only by the user looks like a little more secure way to pass the password

Reply:
Yeah but it's not secure from any point of view, it is better to not have such things @vadmeste